eng/pipelines/stress-http.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+pool:
+  name: Hosted VS2017
+
+variables:
+  _buildConfiguration: Release
+  httpStressProject: 'src/System.Net.Http/tests/StressTests/HttpStress/'
+  httpStressArgs: '-maxExecutionTime 20 -displayInterval 60'
+
+steps:
+- checkout: self
+  clean: true
+  fetchDepth: 1
+  lfs: false
+
+- task: CmdLine@2
+  displayName: Build Corefx
+  inputs:
+    script: |
+      .\build.cmd -ci -configuration $(_buildConfiguration)
+
+- task: PowerShell@2
```

Comment:
There's a short syntax for [inline powershell scripts](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell?view=azure-devops#yaml-snippet). Instead of having to pass down the inputs and declaring the task you can just do:

```yml
- powershell: |
      cd '$(Build.SourcesDirectory)/$(HttpStressProject)'
      # Load testhost sdk in environment
      . .\load-corefx-testhost.ps1 -c $(_buildConfiguration)
      # Run the stress suite
      dotnet run -c $(_buildConfiguration) -- $(httpStressArgs)
  displayName: Run HttpStress
```